Automattic/wp-calypso
# Forms: Remove global form radio styles

#### Changes proposed in this Pull Request

* Remove global form radio styles in favor of specific component-level styling

#### Testing instructions



* Test all the following areas against production and this branch and ensure that they match stylistically and functionally:
    - [ ] NPS Survey "Would you recommend" options (see in devdocs)
    - [ ] Subscription length picker (devdocs)
    - [ ] French domain registration extra info (I couldn't figure out how to find this in the UI)
    - [ ] Devdocs `FormFields` example
    - [ ] Calypso UI Theme picker (in `me/account`)
    - [ ] Cancel auto-renewal and check the survey that happens
    - [ ] Cancel purchase form (go to a recent purchase and trash it and check the modal)
    - [ ] Classic editor Contact form preview (form buttons intentionally `noop`, but add a radio button one and make sure it renders correctly)
    - [ ] WooCommerce paypal settings modal (found in payments settings for store)
    - [ ] WooCommerce stripe settings modal (found in payments settings for store)
    - [ ] Auto capture toggle (I couldn't find this one)
* Ensure all `input type="radio"` have been replaced with `FormRadio`
